docs/payid-cli.md

```diff
@@ -4,6 +4,8 @@ title: PayID CLI
 sidebar_label: PayID CLI
 ---
 
+**NOTES** - This doc will be cut up into tutorial and concepts.
```

Comment:
I suggest concept and reference. Usually a CLI is a (particularly boring) means to an end, so it may come up in the context of other tutorials but anything beyond the most basic of examples probably needs more of a task-oriented context. (e.g. "Here's how to delete old PayIDs using the CLI")

Reply:
I agree, as the Verifiable PayID tutorial actually covers PayID CLI tutorial content anyway.